test/build.info

```diff
@@ -260,7 +260,7 @@ IF[{- !$disabled{tests} -}]
 
   INCLUDE[testutil.o]=..
   INCLUDE[ssl_test_ctx.o]=../include
-  INCLUDE[handshake_helper.o]=../include
+  INCLUDE[handshake_helper.o]=../include ../ssl
```

Comment:
I'm pretty sure I only pulled in -I../ssl for packet_locl.h, since using PACKET_* is a couple orders of magnitude nicer than not using them.  But, I guess it is a slippery slope to have that sitting there, in that something less innocuous could more easily slip in.